### ISSUE TITLE
NTR: Lip: grooving around the lips (Mouth & throat)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7761]
New term request Lip: grooving around the lips (elthat_120822204737): Grooves

**Comment by pnrobinson** [comment]
Is this distinct from [Lip fissure HP:0031250](https://hpo.jax.org/app/browse/term/HP:0031250) ?

**Comment by pnrobinson** [comment]
@MickeySegal -- what disease is associated with this finding? Do you mean a circumferential groove?

**Comment by MickeySegal** [comment]
It sounds different.  An example at https://www.ncbi.nlm.nih.gov/books/NBK100826/ "Multiple episodes of acute photosensitivity may lead to chronic changes of sun-exposed skin (lichenification, leathery pseudovesicles, grooving around the lips) and loss of lunulae of the nails. "

**Comment by pnrobinson** [comment]
Is it this?
https://pubmed.ncbi.nlm.nih.gov/36084634/
(Figure 1A)

**Comment by MickeySegal** [comment]
All I have is the abstract for that paper, but "grooving around the lips" sounds like it is parallel to the lips and notching sounds perpendicular.  But it would be good to get further information from someone who knows this area.

**Comment by pnrobinson** [comment]
@sabrinatoro Could we discuss this term with your colleagues?

**Comment by sabrinatoro** [comment]
From Mary Marazita (Distinguished Professor at the University of Pittsburg, Department of Oral and Craniofacial Sciences):

The term request if for “grooving around the lips”

 

Query:  is this a fissure in the lip?  NO, a fissure in the lip would in the vermillion border.  This grooving around the lips is outside the vermillion border---I have only seen this term in the context of an acquired phenotype due to sun exposure in a photophobic syndrome (eg as in the paper provided).

 

OVERALL:  I can’t comment on whether these terms should be added to HPO since I am unfamiliar with the literature, but I can confirm my interpretations above that these terms do not overlap with our upper lip work.

**Comment by MickeySegal** [comment]
I tried to track down the original references and found a description that seems very confused:
https://www.merckmanuals.com/home/hormonal-and-metabolic-disorders/porphyrias/erythropoietic-protoporphyria-and-x-linked-protoporphyria
"If skin protection is chronically neglected, rough, thickened, and leathery skin (lichenification) may develop, especially over the knuckles. Deep grooves may develop around the mouth (carp mouth)."

**Comment by pnrobinson** [comment]
@MickeySegal 
given that we do not have a precise description of the phenotypic feature, I think it is probably best not to add a term at this time. We can also come back to this if we get more information.
